vmaas_sync/system_culling_test.go

```diff
@@ -143,11 +143,13 @@ func TestCullSystems(t *testing.T) {
 	nToDelete := 4
 	for i := 0; i < nToDelete; i++ {
 		invID := fmt.Sprintf("00000000-0000-0000-0000-000000000de%d", i+1)
+		insightsID := fmt.Sprintf("00000000-0000-0000-0000-000000000de%d", i+1)
```

Comment:
Isn't that `insightsID = invID`?

Reply:
What's insights_id? Isn't it actually inventory_id which is already part of system_platform table?

Reply:
@MichaelMraka maybe it is the case. insights_id was provided by the inventory UI app which is needed for displaying the Inventory table. My assumption was that insights_id is synced when a new system is uploaded/registered. Thus, I have tried to sync this info also together with other system information into the system_platform.  Please have a look at the picture and let me know if it is already in the system_platform as inventory_id

x-special/nautilus-clipboard
copy
file:///home/muslimjon/Pictures/insights_id.png

Reply:
I see, it's yet another different id :( . Inventory_id is referred as 'id' on your picture.

Reply:
So you have it right you need to store it as a new column. Just please for the test use different value than system's inventory_id, e.g "00000000-0000-0000-1111-000000000de%d".